### ISSUE TITLE
#572470082------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1155]
<strong>Source:</strong>`<span class="title" id="b837920e-79fb-d0fc-9e0f-a900c424355f"><slot></slot></span>`

<strong>Element Location: </strong>["calcite-chip[data-name=\"statusCount2\"]","#b837920e-79fb-d0fc-9e0f-a900c424355f"]

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix any of the following:
  Element has insufficient color contrast of 2.94 (foreground color: #ffffff, background color: #35ac46, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

<strong>Tags:</strong> cat.color,wcag2aa,wcag143,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/color-contrast?application=axeAPI
